source/changelog.txt

```diff
@@ -12,6 +12,37 @@
    :depth: 1
    :class: singlecol
 
+v1.0.6
+------
+
+*Released September 14, 2021*
+
+New features in this release:
+
+- You can now run :method:`sh.status()` when connected to a ``mongos``.
+
+- ``db.setSecondaryOk()``, ``mongo.setSecondaryOk()``, and
+  ``rs.secondaryOk()`` methods are reintroduced but **deprecated**.
+  These methods alias to ``mongo.setReadPref()``.
+
+- When you enter multiline input into the shell, single-line
+  ``// comments`` are now preserved as ``/* comments */`` in the
+  history entry.
+
+- Adds a new snippet for resuming tokens.
```

Comment:
“resuming tokens” is not really a thing, and I don’t think I’d note individual snippets here since they can be used independently of the mongosh version

Reply:
Understood. Will remove. Thanks!